tests/examples/test_scripts.py

```diff
@@ -52,23 +52,24 @@ def run_test(filepath):
 @pytest.mark.parametrize(
     "step,file",
     [
-        ("finetuning", "image_classification.py"),
+        # ("finetuning", "image_classification.py"),
         # ("finetuning", "object_detection.py"),  # TODO: takes too long.
         # ("finetuning", "summarization.py"),  # TODO: takes too long.
         ("finetuning", "tabular_classification.py"),
-        ("finetuning", "text_classification.py"),
+        # ("finetuning", "text_classification.py"),
         # ("finetuning", "translation.py"),  # TODO: takes too long.
-        ("predict", "classify_image.py"),
-        ("predict", "classify_tabular.py"),
-        ("predict", "classify_text.py"),
-        ("predict", "image_embedder.py"),
-        ("predict", "summarize.py"),
+        # ("predict", "classify_image.py"),
+        # ("predict", "classify_tabular.py"),
+        # ("predict", "classify_text.py"),
+        # ("predict", "image_embedder.py"),
+        # ("predict", "summarize.py"),
```

Comment:
@tchaton Why is it commented?

Reply:
Because the weights generated are using the new API and the weights were deleted.

Reply:
Got it!